packages/@react-aria/virtualizer/src/ScrollView.tsx

```diff
@@ -184,7 +184,7 @@ export function useScrollView(props: ScrollViewProps, ref: RefObject<HTMLElement
       // adjusted space. In very specific cases this might result in the scrollbars disappearing
       // again, resulting in extra padding. We stop after a maximum of two layout passes to avoid
       // an infinite loop. This matches how browsers behavior with native CSS grid layout.
-      if (!isTestEnv && clientWidth !== dom.clientWidth || clientHeight !== dom.clientHeight) {
```

Comment:
Unrelated to this change, `clientWidth !== dom.clientWidth || clientHeight !== dom.clientHeight` missed a parenthesis for the if condition to make more sense.